caravel/models.py

```diff
@@ -1101,6 +1101,13 @@ def fetch_metadata(self):
                     metric_type='sum',
                     expression="SUM({})".format(quoted)
                 ))
+            if dbcol.sum:
```

Comment:
Do we have tests covering metrics/groupby of db queries? I was trying to look for them but didn't seem to exist core_tests.py